Resources/Textures/_White/Clothing/Head/Hardhats/orange.rsi/meta.json

```
@@ -0,0 +1,42 @@
+{
+  "version": 1,
+  "license": "CC-BY-NC-SA-4.0",
```

Biome (2.1.2)

[error] The key license was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)